packages/kamino-sdk/src/Kamino.ts

```diff
@@ -4590,6 +4591,13 @@ export class Kamino {
       new UpdateReward2Fee(),
       performanceFeeBps
     );
+    let updateMintingMethodToProportionalIx = await getUpdateStrategyConfigIx(
+      strategyAdmin,
+      this._globalConfig,
+      strategy,
+      new UpdateDepositMintingMethod(),
+      new Decimal(1)
```

Comment:
What does this number here refer to? Could you maybe make it a constant so it's easier to understand?

Reply:
done